examples/alphaTestExamples/MultipleServices.md

```diff
@@ -47,20 +50,24 @@ curl -s -o "$BASE/mysql/#1.yaml" "https://raw.githubusercontent.com\
 /{secret,deployment,service}.yaml"
 
 function expectedOutputLine() {
-  test 1 == \
-  $(grep "$@" $OUTPUT/status | wc -l); \
-  echo $?
+  if ! grep -q "$@" "$OUTPUT/status"; then
+    echo -e "${RED}Error: output line not found${NC}"
+    echo -e "${RED}Expected: $@${NC}"
+    exit 1
+  else
+    echo -e "${GREEN}Success: output line found${NC}"
```

Comment:
If you have an idea of how to do that succinctly, I'm all ears. I decided not to re-print the entire output again. And I don't know how to easily get the command run or the line number... Got ideas?

Reply:
FWIW, with every test fatal, we can at least tell which code block caused the failure, if not the exact line.

Reply:
I think the only reasonable way to do this would be to print the text that was expected. But it might be that it clutters  up the output so it isn't much help.